server/service/individualServices/ElasticsearchPreparation.js

```diff
@@ -1,48 +1,76 @@
 const { elasticsearchService, getIndexAliasAsync, operationalStateEnum } = require('onf-core-model-ap/applicationPattern/services/ElasticsearchService');
+const logicalTerminationPoint = require('onf-core-model-ap/applicationPattern/onfModel/models/LogicalTerminationPoint');
```

Comment:
I can see that this file is copied from ApplicationLayerTopology. Kindly please revert this file to original state.

Reply:
Resolved